tests/mock/test_mock_control_closed_loop.py

```diff
@@ -139,14 +139,14 @@ def test_get_demanded_force_axial(self):
         self.control_closed_loop.calc_look_up_forces(59.06)
         demanded_force = self.control_closed_loop.get_demanded_force()
 
-        self.assertAlmostEqual(demanded_force[0], 130.3597559)
-        self.assertAlmostEqual(demanded_force[1], 166.0450977)
+        self.assertAlmostEqual(demanded_force[0], 130.3581281)
```

Comment:
Do you have any idea why the comparison values changed? Depending on the reason it might be best to reduce the precision which you are comparing them.. 

Do you think you could extract **all** these values into a data class or a method in the test class and give them representative names?

Reply:
The values are changed because the original used configuration file in `ts_config_mttcs` does not have the enough digits for the matrix (for the pseudo-inverse). I think the new values here should be more accurate/correct in the math and match the original configuration files by Harris. Thanks!